app/components/public/social-links.js

```diff
@@ -9,5 +10,14 @@ export default Component.extend({
 
   twitterLink: computed('socialLinks.[]', function() {
     return this.socialLinks.findBy('isTwitter', true);
+  }),
+
+  calender: computed( function() {
```

Comment:
Codacy found an issue: [There should be no space after this paren.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6400364)